### PR TITLE
Add recipe for mu_repo 1.4.0

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Comment by nicoddemus** [comment]
Current package does not contain a license file (fabioz/mu-repo#27).

**Comment by nicoddemus** [comment]
All builds have passed, unless someone wants to change something in the recipe we can merge this! 👍

**Comment by ericdill** [comment]
Can you ask on the github repo for the author to push a tag to note which commit is 1.4.0?

**Comment by nicoddemus** [comment]
Thanks!

Done: fabioz/mu-repo#28

**Comment by ericdill** [comment]
Thanks @nicoddemus

File: recipes/mu-repo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "mu_repo" %}
```
Comment (jakirkham) [inline recipes/mu-repo/meta.yaml]: Directory needs to be renamed to match the recipe for conversion of this to complete. Could you please send a follow-up PR to correct this, @nicoddemus? Feel free to add `[ci skip]` to the commit message as we already know the recipe works. Thanks.

Comment (nicoddemus) [inline recipes/mu-repo/meta.yaml]: Sure, no problem (this whole deal with `_` and `-` is confusing in this package I admit).

https://github.com/conda-forge/mu_repo-feedstock does not seem to have been rendered though. @jakirkham do you mean for me to open another PR?

Comment (ericdill) [inline recipes/mu-repo/meta.yaml]: @nicoddemus, @ocefpaf already took care of the rename: https://github.com/conda-forge/staged-recipes/pull/2726

Comment (nicoddemus) [inline recipes/mu-repo/meta.yaml]: Ahh nice, thanks!

**Pull request merged**